### PR TITLE
Add listeners for NavigationDrawer menu items

### PR DESCRIPTION
**Pull request by ermel272**
### Description
Adds logic to the Navigation Drawer's menu items.

### Changelog
- Fix race condition that occurred at end of twitter auth flow
- Added logic to nav drawer menu items
- Put migrate app name back in tweetfeedactivity action bar

### Addresses Issues

### CODE REVIEW

**Comment by bazadactyl** [comment]
Good stuff. Looking forward to trying out these changes.

File: app/src/main/java/com/dankideacentral/dic/authentication/OAuthWebViewFragment.java [hunk 1]
```diff
@@ -26,28 +29,31 @@
...
-                        intent = new Intent(getActivity().getApplicationContext(),
-                                TweetFeedActivity.class);
-                    } else {
-                        intent = new Intent(getActivity().getApplicationContext(),
-                                SearchActivity.class);
-                    }
-
-                    startActivity(intent);
+                            getActivity().getApplicationContext(),
+                            new Handler.Callback() {
```
Comment (srowhani) [inline app/src/main/java/com/dankideacentral/dic/authentication/OAuthWebViewFragment.java:44]: damn, looks fresh

**Review by srowhani — APPROVED** [review]
🔥

File: app/src/main/java/com/dankideacentral/dic/authentication/OAuthWebViewFragment.java (hunk 1 above)
Comment (ermel272) [reply app/src/main/java/com/dankideacentral/dic/authentication/OAuthWebViewFragment.java:44]: Thanks for telling me about the Callback, learned something new ahyuehehehehehehe

**Pull request merged**